app/controllers/subscriptions_controller.rb

```diff
@@ -80,7 +80,8 @@ def destroy
   private
 
   def notify_new_subscription(subscription)
-    subscription.update(recaptcha_score: recaptcha_reply&.dig("score"))
+    # subscription.update(recaptcha_score: recaptcha_reply&.dig("score"))
+    subscription.save!
```

Comment:
`subscription` was being persisted by the update method. As I have commented it out here I needed to call save on the subscription to persist it.